### PR TITLE
added timeline translation for arr

### PR DESCRIPTION
**Pull request by Akurosia**

### CODE REVIEW

**Comment by mooondark** [comment]
Hello,
why did you remove the space in `'Engage!': 'À l\'attaque !',` because, there is a space, really.

**Comment by Akurosia** [comment]
This seams to be an error of my script interpreting the `À l'attaque<Indent/>!` as `À l'attaque!` since the `<Indent/>` is not always equals a single space.
Thanks for pointing this out, will update my PRs today/night

File: ui/raidboss/data/02-arr/raid/t5.txt [hunk 1]
```diff
@@ -38,7 +38,7 @@ hideall "--sync--"
 400.0 "Divebomb" sync /:Twintania:5B0:/ window 400,0
 407.9 "Divebomb" sync /:Twintania:5B0:/
 415.6 "Divebomb" sync /:Twintania:5B0:/
-419.6 "1x Ascelpius add"
+419.6 "1x Asclepius add"
```
Comment (quisquous) [inline ui/raidboss/data/02-arr/raid/t5.txt]: Whoops, thanks.

**Comment by quisquous** [comment]
Thanks!

**Pull request merged**